src/maingoblint.ml

```diff
@@ -256,6 +253,7 @@ let preprocess_files () =
   let rec preprocess_arg_file = function
     | filename when Filename.basename filename = "Makefile" ->
       let comb_file = MakefileUtil.generate_and_combine filename ~all_cppflags in
+      (* The resulting dependency information will not be correctly tracked, because the merge happens in cilly *)
```

Comment:
I suppose this comment isn't necessary anymore since this would just make all the combined files dependencies of `comb_file`, which is right.